sdk/reservations/Azure.ResourceManager.Reservations/src/Generated/Models/PurchaseRequestContent.cs

```diff
@@ -79,7 +79,7 @@ public string SkuName
         /// <summary> Type of the Applied Scope. </summary>
         public AppliedScopeType? AppliedScopeType { get; set; }
         /// <summary> List of the subscriptions that the benefit will be applied. Do not specify if AppliedScopeType is Shared. </summary>
-        public IList<string> AppliedScopes { get; }
+        public IList<string> AppliedScopes { get; set; }
```

Comment:
For non-required property, we will initialize it using `ChangeTrackingList<T>` in ctor, then you can use `list = {}` or `list.Add()` to set its value, so the setter of this property is removed. This is by design, please don't add the setter.

Reply:
Reverted back